frontend/public/components/resource-quota.jsx

```diff
@@ -42,10 +47,20 @@ const clusterResourceQuotaMenuActions = [
 
 const isClusterQuota = (quota) => !quota.metadata.namespace;
 
-const quotaKind = (quota) =>
-  isClusterQuota(quota)
-    ? referenceForModel(ClusterResourceQuotaModel)
-    : referenceForModel(ResourceQuotaModel);
+const resourceQuotaReference = referenceForModel(ResourceQuotaModel);
+const clusterQuotaReference = referenceForModel(ClusterResourceQuotaModel);
+const appliedClusterQuotaReference = referenceForModel(AppliedClusterResourceQuotaModel);
+
+const quotaKind = (quota) => {
```

Comment:
Not new, but `quotaKind` is a misleading name. Is there a reason we can't just use `referenceFor`?